code/datums/components/transforming.dm

```diff
@@ -152,8 +152,8 @@
  */
 /datum/component/transforming/proc/default_transform_message(obj/item/source, mob/user)
 	if(user)
-		source.balloon_alert(user, "[active ? "enabled" : "disabled"] [source]")
-	playsound(user ? user : source.loc, 'sound/weapons/batonextend.ogg', 50, TRUE)
+		source.balloon_alert(user, "[active ? "enabled" : "disabled"]")
```

Comment:
this is the line where you cleaned up the balloon alerts that made me nit everyting else, which makes sense since this is a generic component and not anything else. you can probably ignore the other balloon alert change requests if you want to but i just thought i'd point it out